Examples/src/components/Examples/Charts2D/StylingAndTheming/UsePointMarkers/drawExample.ts

```diff
@@ -125,7 +124,7 @@ export const drawExample = async (rootElement: string | HTMLDivElement) => {
     );
 
     // Add a scatter series with Custom Image using SpritePointMarker
-    const imageBitmap = await createImageAsync(customPointImage);
+    const imageBitmap = await createImageAsync(emojiUrls);
```

Comment:
Minor hint: no need to rename emojiUrls every image path to `emojiUrls`.
Just leave it named as is so it makes sense in the context...